va/va_dec_hevc.h

```diff
@@ -194,6 +194,154 @@ typedef struct  _VAPictureParameterBufferHEVC
+            uint32_t        palette_mode_enabled_flag                       : 1;
+            /*  controls the presence and inference of the use_integer_mv_flag syntax
+             *  in slice segment header that specifies the resolution of motion
+             *  vectors for inter prediction.
+             */
+            uint32_t        motion_vector_resolution_control_idc            : 2;
+            /*  specifies that the intra boundary filtering process is
+             *  disabled or not for intra prediction.
+             */
+            uint32_t        intra_boundary_filtering_disabled_flag          : 1;
+            /*  specifies that an adaptive colour transform may be applied
+             *  to the residual in the decoding process.
+             */
+            uint32_t        residual_adaptive_colour_transform_enabled_flag : 1;
+
+            uint32_t        pps_slice_act_qp_offsets_present_flag           : 1;
+
+            uint32_t        reserved_bits                                   : 25;
+        } bits;
+        uint32_t            value;
+    } screen_content_pic_fields;
+
+    /*  specifies the maximum allowed palette size. */
+    uint8_t                 palette_max_size;
+    /*  Correspond to HEVC syntax elements of the same names.
+     *  It specifies the difference between the maximum allowed palette
+     *  predictor size and the maximum allowed palette size.
+     *  App needs to enforce that the variable PaletteMaxPredictorSize,
+     *  which is derived as follows:
+     *  PaletteMaxPredictorSize = palette_max_size + delta_palette_max_predictor_size
+     *  should have a value range of [0..128].
+     */
+    uint8_t                 delta_palette_max_predictor_size;
+    /** \brief Size of initial palette predictor.
+     *  It is derived from pps_num_palette_predictor_initializer or
+     *  sps_num_palette_predictor_initializer_minus1.
+     *  Details in HEVC SCC spec section 9.3.2.3.
+     */
+    uint8_t                 predictor_palette_size;
+    /** \brief Palette predictor initializer.
+     *  It is derived from pps_palette_predictor_initializers[][]
+     *  or sps_palette_predictor_initializers[][].
+     *  Details in HEVC SCC spec section 9.3.2.3.
+     */
+    uint16_t                predictor_palette_entries[3][128];
+    /*  are used to determine the offsets that are applied to the
+     *  quantization parameter values for the luma, Cb and Cr
+     *  components, respectively.
+     */
+    int8_t                  pps_act_y_qp_offset_plus5;
+    int8_t                  pps_act_cb_qp_offset_plus5;
+    int8_t                  pps_act_cr_qp_offset_plus3;
+} VAPictureParameterBufferHEVCScc;
+
+/**
+ * \brief HEVC Decoding Picture Parameter Buffer Structure including Extensions
+ *
+ * This structure conveys picture level HEVC parameters including basic version 1
+ * and range extension and screen content extension.
+ * The data buffer should be sent once per frame.
+ *
```

Comment:
@xhaihao yes, it dose. for hevc scc decoding, VAPictureParameterBufferHEVC, VAPictureParameterBufferHEVCRext, VAPictureParameterBufferHEVCScc must be send together and parsed together. and for hevc range extension decoding, only VAPictureParameterBufferHEVC, VAPictureParameterBufferHEVCRext should be sent together.

Reply:
@xhaihao yes, it will be sent, but will not be parsed. if separate it from VAPictureParameterBufferHEVCExtension , you may need another structure like VAPictureParameterBufferHEVCScc that contains all picture parameters. we consider hevc scc profile as hevc range extension as well, so it makes sense to add scc parameters in VAPictureParameterBufferHEVCExtension.